### PR TITLE
fixed the strings display issue

### PR DESCRIPTION
**Pull request by phoenix-23**
Only the first word of all string objects was being displayed on the frontend side. This minor issue is fixed now.

### CODE REVIEW

File: src/user/templates/hcadmin_profile.html [hunk 1]
```diff
@@ -110,7 +110,7 @@ <h1 class="h2">View Profile</h1>
                     </div>
                     <div class="form-group col-md-6">
                         <label for="address">Address</label>
-                        <input type="text" class="form-control" id="address" value={{hcadmin.user.address}} disabled>
+                        <input type="text" class="form-control" id="address" value="{{hcadmin.user.address}}" disabled>
                     </div>
                     <div class="form-group col-md-6">
                         <label for="designation">Designation</label>
```
Comment (Gaukm01) [inline src/user/templates/hcadmin_profile.html:116]: Make the changes for all types of fields with type "text", that would not restrict any part to a single word. like for Designation & Department fields too.

**Review by Gaukm01 — COMMENTED** [review]
@phoenix-23  I think we should add "..." for all the text fields and not assume that the field will have only one word so no need to add this, even for Dates, etc.

File: src/user/templates/hcadmin_profile.html (hunk 1 above)
Comment (phoenix-23) [reply src/user/templates/hcadmin_profile.html:116]: ok, will do so

**Comment by sanchit-sinha** [comment]
Kindly update this value in lab name(``Lab Name Field missing``) to it's lab name!
https://github.com/shoryak/hc_reimbursement/blob/b04d6261ec8066ac366aff9ee7a9bc5433083eba/src/user/templates/patient_dashboard.html#L188

Please do so at every place wherever it is present

**Review by Gaukm01 — APPROVED** [review]

**Pull request merged**